cluster/stone-prd-rh01/tenants/rhtap-build-tenant/integration_test_scenario.yaml

```diff
@@ -89,3 +89,25 @@ spec:
       - name: pathInRepo
         value: pipelines/enterprise-contract.yaml
     resolver: git
+---
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  name: build-task-dockerfile-e2e-test
+  namespace: rhtap-build-tenant
+spec:
+  application: build-tasks-dockerfiles
+  contexts:
+    - description: E2E testing
+      name: application
+  params:
+    - name: param1
```

Comment:
This probably doesn't need to be here.

What about the SNAPSHOT param? Does the integration service pass that automatically?

Reply:
Good catch. Its not needed. As per my understanding, `SNAPSHOT` param is not required in IntegrationTestScenario, it is required in the integration pipeline, that is already added.